File: src/test/java/be/cegeka/orderit/service/eventstore/api/InternalAnalysisRemovedEventTest.java
package be.cegeka.orderit.service.eventstore.api;

public class InternalAnalysisRemovedEventTest extends BaseInternalAnalysisEventTest {

    @Override
    protected AbstractInternalAnalysisEvent item() {
        final InternalAnalysisAggregateId aggregateIdentifier = new InternalAnalysisAggregateId("safdghjk");
        return new InternalAnalysisRemovedEvent(aggregateIdentifier, AnalysisNumber.from(45), "user");
    }
}

File: src/test/kotlin/be/cegeka/orderit/service/eventstore/api/InternalAnalysisRemovedEventTest.kt
package be.cegeka.orderit.service.eventstore.api

import be.cegeka.orderit.service.eventstore.api.AnalysisNumber.Companion.from

class InternalAnalysisRemovedEventTest : BaseInternalAnalysisEventTest() {

    override fun item(): AbstractInternalAnalysisEvent {
        val aggregateIdentifier = InternalAnalysisAggregateId("safdghjk")
        return InternalAnalysisRemovedEvent(aggregateIdentifier, from(45), "user")
    }
}